bot/cogs/timeleft.py

```diff
@@ -0,0 +1,23 @@
+from discord.ext import commands
+from dateutil.relativedelta import relativedelta
+import logging
+import aiohttp
+import discord
+import datetime
+
+bot = commands.Bot(command_prefix='.', description='pr bot')
+
+@bot.command()
+async def timeleft():

```

Comment:
Needs to [accept a `Context` input (generally as `ctx`)](https://discordpy.readthedocs.io/en/rewrite/ext/commands/commands.html) so the bot knows where to respond.